config/settings.yml

```diff
@@ -18,13 +18,12 @@ logingov:
   inherited_proofing_redirect_uri: http://localhost:3000/inherited_proofing/callback
 
 idme:
-  client_id: ef7f1237ed3c396e4b4a2b04b608a7b1
-  client_secret: ae657fd2b253d17be7b48ecdb39d7b34
+  client_id: dde0b5b8bfc023a093830e64ef83f148
+  client_secret: e745319288bd005c9fca7d8164f302c7
   redirect_uri: http://localhost:3000/sign_in/idme/callback
   client_cert_path: spec/fixtures/sign_in/oauth.crt
   client_key_path: spec/fixtures/sign_in/oauth.key
   oauth_url: https://api.idmelabs.com
-  oauth_public_key: spec/fixtures/idme/idme_oauth_pub.pem
```

Comment:
Did you mean to make these changes?

Reply:
No. Just tried to merge the latest changes from master into my current branch.